src/applications/personalization/dashboard-2/tests/unit/healthcare-appointments.unit.spec.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
+import reducers from '~/applications/personalization/dashboard/reducers';
+import HealthCare from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
+import {
+  farFutureAppointments,
+  upcomingVAAppointment,
+} from '~/applications/personalization/dashboard-2/utils/appointments';
+
+describe('HealthCare component', () => {
+  let view;
+  let initialState;
+
+  describe('when the user has an appointment scheduled within the next 30 days', () => {
+    it('should render "Next appointment"', async () => {
+      window.VetsGov = { pollTimeout: 1 };
+      initialState = {
+        user: {
+          profile: {
+            services: [],
+          },
+        },
+        health: {
+          appointments: {
+            fetching: false,
+            data: upcomingVAAppointment,
+          },
+          msg: {
+            folders: {
+              data: {
+                currentItem: {
+                  attributes: {
+                    unreadCount: null,
+                  },
+                  fetching: false,
+                },
+              },
+              ui: {
+                nav: {
+                  foldersExpanded: false,
+                  visible: false,
+                },
+              },
+            },
+          },
+        },
+      };
+
+      view = renderInReduxProvider(<HealthCare dataLoadingDisabled />, {
```

Comment:
the more I see the `dataLoadingDisabled` used the more it makes me uncomfortable. I'm going to look into making it as easy as possible for us to mock basic API calls with `msw` :)